src/mon/MDSMonitor.cc

```diff
@@ -988,12 +988,16 @@ bool MDSMonitor::preprocess_command(MonOpRequestRef op)
       ostringstream ess;
```

Comment:
the ess here is unused, however, it contains far more detailed information about each invalid id than what you will have with a simple list of unmatched input ids.